Simonyi/Non-Standard-Operations/MTCS/MTPtg/MTPtg-Configuration-for-MTRot-MTMount.rst

```diff
@@ -0,0 +1,194 @@
+or you want to revert the change and **include** the mount.
+
+
+.. _MTMTPtg-Configuration-for-MTRotator-and-MTMount-Procedure-Procedure-Steps:
+Procedure Steps
+===============
+
+.. warning:: 
+    
+    Announce through the Slack channel *#summit-simonyi* that the component is not available, and you are about to change the configuration.
+    
+    The Commissioning Scientists on shift will change the version of ``MTMount`` CSC in *ArgoCD*; this can deal with tracking without said component.
+
+        The Commissioning Scientists  may request to send the ``MTMount`` to **OFFLINE**. this will change the ``MTMount`` CSC version to **mtmount-ccw-only**. 
+..
+
+
+
+Steps
+-----
+
+1. Issue the :file:`set_summary_state.py` script in LOVE to change the status of ``MTPtg`` to ``STANDBY`` with the following configuration
+
+    .. code-block::
+        :caption: set_summary_state.py
+
+             data:
+                 -
+                   - MTPtg 
+                   - STANDBY
+..
+
+
+2. Find the name of the **pod** where the ``MTPtg`` is running. 
+   
+    From your terminal, run the following command which list all the **pods** related to the *<namespace>* maintel:
+
+    .. prompt:: bash
+
+     kubectl --kubeconfig=${HOME}/.kube/yagan.yaml get pod -o=custom-columns=NAME:.metadata.name,STATUS:.status.phase,NODE:.spec.nodeName -n maintel
+
+    ..
+
+    .. figure:: ./_static/1.png
+      :width: 950px
+      :height: 165px
+      :name: Your figure
+
+      In this particular case the name of the ``MTPtg`` **pod** is **mtptg-djhpv**.
+    ..  
+
+3. Connect to the ``MTPtg`` **pod mtptg-djhjv** within the *<namespace>* maintel. 
+
+    The command will open a terminal within the pod.
+
+    .. prompt:: bash
+
+     kubectl --kubeconfig=${HOME}/.kube/yagan.yaml exec --stdin --tty mtptg-djhpv -n maintel -- /bin/bash
+
+    ..
+   
+    .. figure:: ./_static/2.png
+        :width: 900px
+        :height: 65px
+
+    ..
+
+4. **Move to configuration directory,** the configuration files are one level up. The directory contains the configuration files :file:`MTPtg.info` (:file:`ATPtg.info` for AuxTel) and the pointing models :file:`mt.mod` files (:file:`at.mod` for AuxTel). 
+
+    At startup, the pointing component loads by default the pointing model :file:`mt.mod` file and the :file:`MTPtg.info` (equivalent for AuxTel).
+
+    .. prompt:: bash
+
+     [saluser@podname] cd /home/saluser/repos/ts_pointing_common/install/data
+
+    ..
+    
+    .. figure:: ./_static/3.png
+       :width: 900px
+       :height: 420px
+        
+    ..
+
+5. To edit the :file:`MTPtg.info` file use a text editor such as *vi*. 
+
+    5.1. ``MTRotator``: The **disable_rotator** line of the :file:`MTPtg.info` file contains the parameter you need to change. It reflects whether the ``MTRotator`` is monitored in the tracking.
```

Comment:
Consider rewriting as 
'Edit the disable_rotator parameter in the MTPtg.info file
* set 1: rotator will be ignored and not be commanded by the MTPtg component (disabled)
* set 0: rotator will be considered (enabled) '